src/proposals/proposalTypes/HybridProposal.sol

```diff
@@ -479,6 +536,22 @@ abstract contract HybridProposal is
                 bytes[] memory payloads
             ) = getTargetsPayloadsValues(addresses);
 
+            if (baseActions.length != 0) {
```

Comment:
we do a lot of validations on this function, I suggest we split this into 3 functions to avoid mistakes:
1. base actions validations
2. moonbeam actions validations
3. validations that apply to both